apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -275,15 +276,15 @@ export const EventTypeList = ({
     },
   });
 
-  async function moveEventType(index: number, increment: 1 | -1) {
+  async function moveEventType(index: number, increment: number, enableAnimation: boolean) {
+    enableAnimations(enableAnimation);
     const newList = [...types.map(normalizeEventType)];
-
     const type = types[index];
-    const tmp = types[index + increment];
-    if (tmp) {
-      newList[index] = normalizeEventType(tmp);
-      newList[index + increment] = normalizeEventType(type);
-    }
+
+    newList.splice(index, 1);
+
```

Comment:
Initially thought this was causing animation issues, but isn't the case. lmk if this should be reverted